oceannavigator/__init__.py

```diff
@@ -37,6 +39,19 @@ def create_app(testing: bool = False):
     app.config.from_pyfile('oceannavigator.cfg', silent=False)
     app.config.from_envvar('OCEANNAVIGATOR_SETTINGS', silent=True)
     app.testing = testing
+
+    if testing:
+        # Override cache dirs when testing
+        # to avoid test files being cached
+        # in production cache
+        cache_dir = tempfile.mkdtemp()
+        tile_dir = tempfile.mkdtemp()
+        app.config.update(
+            CACHE_DIR=cache_dir,
+            TILE_CACHE_DIR=tile_dir
+        )
+        print(f"{ASCIITerminalColors.WARNING}[Warning] -- Cached files will NOT be cleaned after tests complete: {cache_dir} AND {tile_dir}{ASCIITerminalColors.ENDC}")
+
```

Comment:
What this looks like:
![image](https://user-images.githubusercontent.com/5572045/124395766-7c3ba200-dce0-11eb-813f-f7fc2ee49c97.png)